Sources/Build/llbuild.swift

```diff
@@ -151,47 +172,59 @@ public struct LLBuildManifestGenerator {
             }
         }
 
+        var buildTarget = Target(name: target.target.targetName)
```

Comment:
I think you need to change the names of the `main` and `test` targets, because someone could create a SwiftPM target called `main` or `test`, and then it will produce an invalid manifest. Worth a test case, maybe.